database/database.go

```diff
@@ -271,13 +268,72 @@ func (t *Txn) ChannelPage(page uint,
 
 }
 
-// PutChannel inserts a (descriptor, channel) in the database.
+// pageRange contains the start (inclusive) and end (exclusive) index of a
+// pagination.
+type pageRange struct {
+	from uint
```

Comment:
I'd rename `to` and `from` to `start` and `end` (which would be also consistent with the documentation).